pkgs/top-level/python-packages.nix

```diff
@@ -14060,6 +14070,7 @@ self: super: with self; {
 
   torchWithRocm = self.torch.override {
     magma = pkgs.magma-hip;
+    openai-triton = self.openai-triton;
```

Comment:
no-op? Also I wonder what happens if `config.cudaSupport = true`, because then `openai-triton = openai-triton-cuda` I guess?

Reply:
Should I make an explicit -rocm variant then?

Reply:
Wait no, I don't think I can do that.
It would have to be something like ``openai-triton-no-cuda`` instead.

Reply:
Well, seeing how we already got `torchWithoutCuda` for the same purpose, couldn't hurt too much (and we can always remove it later)